diff -dur wine-1.8.orig/loader/wine.desktop wine-1.8/loader/wine.desktop
--- wine-1.8.orig/loader/wine.desktop	2015-12-19 16:57:44.000000000 +0100
+++ wine-1.8/loader/wine.desktop	2016-01-09 18:22:26.828643807 +0100
@@ -24,6 +24,7 @@
 Name[hr]=Wine - dizač Windows programa
 Name[he]=Wine — מריץ תכניות Windows
 Name[ja]=Wine Windowsプログラムローダー
+Comment=Launch Windows Program with Wine
 Exec=wine start /unix %f
 MimeType=application/x-ms-dos-executable;application/x-msi;application/x-ms-shortcut;
 Icon=wine
