diff -dur wine-1.6.orig/tools/wine.desktop wine-1.6/tools/wine.desktop
--- wine-1.6.orig/tools/wine.desktop	2013-07-18 19:57:36.000000000 +0200
+++ wine-1.6/tools/wine.desktop	2013-10-26 17:34:16.255593785 +0200
@@ -24,6 +24,7 @@
 Name[hr]=Wine - dizač Windows programa
 Name[he]=Wine — מריץ תכניות Windows
 Name[ja]=Wine Windowsプログラムローダー
+Comment=Launch Windows Program with Wine
 Exec=wine start /unix %f
 MimeType=application/x-ms-dos-executable;application/x-msi;application/x-ms-shortcut;
 Icon=wine
