--- wine-0.9.28/configure.ac.orig	2006-12-22 17:17:47.000000000 +0100
+++ wine-0.9.28/configure.ac	2006-12-24 03:10:15.504363300 +0100
@@ -194,6 +194,7 @@
 	machine/soundcard.h \
 	mntent.h \
 	ncurses.h \
+	ncurses/ncurses.h \
 	netdb.h \
 	netinet/in.h \
 	netinet/in_systm.h \
@@ -519,7 +520,7 @@
 
 dnl **** Check which curses lib to use ***
 CURSESLIBS=""
-if test "$ac_cv_header_ncurses_h" = "yes"
+if test "$ac_cv_header_ncurses_h" = "yes" || test "$ac_cv_header_ncurses_ncurses_h" = "yes"
 then
     AC_CHECK_LIB(ncurses,waddch,[CURSESLIBS="-lncurses"])
 elif test "$ac_cv_header_curses_h" = "yes"
--- wine-0.9.13/programs/wineconsole/curses.c.old	2006-05-19 12:01:34.492841168 +0200
+++ wine-0.9.13/programs/wineconsole/curses.c	2006-05-19 12:04:32.524776200 +0200
@@ -37,6 +37,8 @@
 #include <stdlib.h>
 #ifdef HAVE_NCURSES_H
 # include <ncurses.h>
+#elif defined(HAVE_NCURSES_NCURSES_H)
+# include <ncurses/ncurses.h>
 #elif defined(HAVE_CURSES_H)
 # include <curses.h>
 #endif
@@ -57,9 +59,9 @@
 
 #define PRIVATE(data)   ((struct inner_data_curse*)((data)->private))
 
-#if defined(HAVE_CURSES_H) || defined(HAVE_NCURSES_H)
+#if defined(HAVE_CURSES_H) || defined(HAVE_NCURSES_H) || defined(HAVE_NCURSES_NCURSES_H)
 
-#ifdef HAVE_NCURSES_H
+#if defined(HAVE_NCURSES_H) || defined(HAVE_NCURSES_NCURSES_H)
 # define CURSES_NAME "ncurses"
 #else
 # define CURSES_NAME "curses"
@@ -115,7 +117,7 @@
 
 static BOOL WCCURSES_bind_libcurses(void)
 {
-#ifdef HAVE_NCURSES_H
+#if defined(HAVE_NCURSES_H) || defined(HAVE_NCURSES_NCURSES_H)
     static const char ncname[] = SONAME_LIBNCURSES;
 #else
     static const char ncname[] = SONAME_LIBCURSES;
