--- wine-20030911/dlls/ntdll/cdrom.c.orig	2003-09-06 01:08:34.000000000 +0200
+++ wine-20030911/dlls/ntdll/cdrom.c	2003-10-01 23:13:13.920825872 +0200
@@ -361,8 +361,9 @@
 #if defined(linux)
     {
         struct stat st;
+	int dummy;
 #ifdef SG_EMULATED_HOST
-        if (ioctl(fd, SG_EMULATED_HOST) != -1) {
+        if (ioctl(fd, SG_EMULATED_HOST, &dummy) != -1) {
             FIXME("not implemented for true scsi drives\n");
             return 0;
         }
