--- wine-1.3.9/server/Makefile.in~	2010-12-10 20:33:07.000000000 +0100
+++ wine-1.3.9/server/Makefile.in	2010-12-10 23:38:49.953335249 +0100
@@ -54,8 +54,8 @@
 INSTALLDIRS = \
 	$(DESTDIR)$(bindir) \
 	$(DESTDIR)$(mandir)/man$(prog_manext) \
-	$(DESTDIR)$(mandir)/de.UTF-8/man$(prog_manext) \
-	$(DESTDIR)$(mandir)/fr.UTF-8/man$(prog_manext)
+	$(DESTDIR)$(mandir)/de/man$(prog_manext) \
+	$(DESTDIR)$(mandir)/fr/man$(prog_manext)
 
 all: $(PROGRAMS)
 
@@ -71,8 +71,8 @@
 	$(INSTALL_PROGRAM) wineserver-installed $(DESTDIR)$(bindir)/wineserver
 
 install-man-pages:: $(EXTRA_MANPAGES) $(INSTALLDIRS)
-	$(INSTALL_DATA) wineserver.de.man $(DESTDIR)$(mandir)/de.UTF-8/man$(prog_manext)/wineserver.$(prog_manext)
-	$(INSTALL_DATA) wineserver.fr.man $(DESTDIR)$(mandir)/fr.UTF-8/man$(prog_manext)/wineserver.$(prog_manext)
+	$(INSTALL_DATA) wineserver.de.man $(DESTDIR)$(mandir)/de/man$(prog_manext)/wineserver.$(prog_manext)
+	$(INSTALL_DATA) wineserver.fr.man $(DESTDIR)$(mandir)/fr/man$(prog_manext)/wineserver.$(prog_manext)
 
 uninstall::
 	$(RM) $(DESTDIR)$(bindir)/wineserver
--- wine-1.3.9/tools/Makefile.in~	2010-12-10 20:33:07.000000000 +0100
+++ wine-1.3.9/tools/Makefile.in	2010-12-10 23:39:44.986668581 +0100
@@ -32,8 +32,8 @@
 	$(DESTDIR)$(datadir)/applications \
 	$(DESTDIR)$(datadir)/wine \
 	$(DESTDIR)$(mandir)/man$(prog_manext) \
-	$(DESTDIR)$(mandir)/de.UTF-8/man$(prog_manext) \
-	$(DESTDIR)$(mandir)/fr.UTF-8/man$(prog_manext)
+	$(DESTDIR)$(mandir)/de/man$(prog_manext) \
+	$(DESTDIR)$(mandir)/fr/man$(prog_manext)
 
 UPDATE_DESKTOP_DATABASE = update-desktop-database
 
@@ -75,8 +75,8 @@
 	$(INSTALL_SCRIPT) $(srcdir)/winemaker $(DESTDIR)$(bindir)/winemaker
 
 install-man-pages:: $(EXTRA_MANPAGES) $(INSTALLDIRS)
-	$(INSTALL_DATA) winemaker.de.man $(DESTDIR)$(mandir)/de.UTF-8/man$(prog_manext)/winemaker.$(prog_manext)
-	$(INSTALL_DATA) winemaker.fr.man $(DESTDIR)$(mandir)/fr.UTF-8/man$(prog_manext)/winemaker.$(prog_manext)
+	$(INSTALL_DATA) winemaker.de.man $(DESTDIR)$(mandir)/de/man$(prog_manext)/winemaker.$(prog_manext)
+	$(INSTALL_DATA) winemaker.fr.man $(DESTDIR)$(mandir)/fr/man$(prog_manext)/winemaker.$(prog_manext)
 
 uninstall::
 	$(RM) $(DESTDIR)$(datadir)/wine/wine.inf \
--- wine-1.3.22/loader/Makefile.in.orig	2011-06-10 20:08:11.000000000 +0200
+++ wine-1.3.22/loader/Makefile.in	2011-06-12 17:41:34.984957008 +0200
@@ -24,9 +24,9 @@
 INSTALLDIRS = \
 	$(DESTDIR)$(bindir) \
 	$(DESTDIR)$(mandir)/man$(prog_manext) \
-	$(DESTDIR)$(mandir)/de.UTF-8/man$(prog_manext) \
-	$(DESTDIR)$(mandir)/fr.UTF-8/man$(prog_manext) \
-	$(DESTDIR)$(mandir)/pl.UTF-8/man$(prog_manext)
+	$(DESTDIR)$(mandir)/de/man$(prog_manext) \
+	$(DESTDIR)$(mandir)/fr/man$(prog_manext) \
+	$(DESTDIR)$(mandir)/pl/man$(prog_manext)
 
 all: $(MAIN_BINARY) wine-installed $(EXTRA_BINARIES)
 
@@ -53,9 +53,9 @@
 	$(INSTALL_PROGRAM) wine-installed $(DESTDIR)$(bindir)/$(MAIN_BINARY)
 
 install-man-pages:: $(EXTRA_MANPAGES) $(INSTALLDIRS) dummy
-	$(INSTALL_DATA) wine.de.man $(DESTDIR)$(mandir)/de.UTF-8/man$(prog_manext)/wine.$(prog_manext)
-	$(INSTALL_DATA) wine.fr.man $(DESTDIR)$(mandir)/fr.UTF-8/man$(prog_manext)/wine.$(prog_manext)
-	$(INSTALL_DATA) wine.pl.man $(DESTDIR)$(mandir)/pl.UTF-8/man$(prog_manext)/wine.$(prog_manext)
+	$(INSTALL_DATA) wine.de.man $(DESTDIR)$(mandir)/de/man$(prog_manext)/wine.$(prog_manext)
+	$(INSTALL_DATA) wine.fr.man $(DESTDIR)$(mandir)/fr/man$(prog_manext)/wine.$(prog_manext)
+	$(INSTALL_DATA) wine.pl.man $(DESTDIR)$(mandir)/pl/man$(prog_manext)/wine.$(prog_manext)
 
 uninstall::
 	-cd $(DESTDIR)$(bindir) && $(RM) $(MAIN_BINARY) $(EXTRA_BINARIES)
