diff -durN -x '*~' -x '*.orig' wine-1.7.18.orig/configure.ac wine-1.7.18/configure.ac
--- wine-1.7.18.orig/configure.ac	2014-05-02 20:15:48.000000000 +0200
+++ wine-1.7.18/configure.ac	2014-05-03 13:18:36.421083206 +0200
@@ -443,6 +443,7 @@
 	machine/sysarch.h \
 	mntent.h \
 	ncurses.h \
+	ncurses/ncurses.h \
 	netdb.h \
 	netinet/in.h \
 	netinet/in_systm.h \
@@ -1354,9 +1355,12 @@
 
 dnl **** Check which curses lib to use ***
 CURSES_LIBS=""
-if test "$ac_cv_header_ncurses_h" = "yes"
+if test "$ac_cv_header_ncurses_h" = "yes" || test "$ac_cv_header_ncurses_ncurses_h" = "yes"
 then
     WINE_CHECK_SONAME(ncurses,waddch,[CURSES_LIBS="-lncurses"])
+    if test "x$ac_cv_lib_soname_ncurses" = "x"; then
+        WINE_CHECK_SONAME(ncursesw,waddch,[CURSESLIBS="-lncursesw"])
+    fi
 elif test "$ac_cv_header_curses_h" = "yes"
 then
     WINE_CHECK_SONAME(curses,waddch,[CURSES_LIBS="-lcurses"])
@@ -1365,8 +1369,8 @@
 LIBS="$LIBS $CURSES_LIBS"
 AC_CHECK_FUNCS(mousemask)
 LIBS="$ac_save_LIBS"
-WINE_NOTICE_WITH(curses,[test "x$ac_cv_lib_soname_curses$ac_cv_lib_soname_ncurses" = "x"],
-                 [lib(n)curses ${notice_platform}development files not found, curses won't be supported.])
+WINE_NOTICE_WITH(curses,[test "x$ac_cv_lib_soname_curses$ac_cv_lib_soname_ncurses$ac_cv_lib_soname_ncursesw" = "x"],
+                 [lib(n)curses(w) ${notice_platform}development files not found, curses won't be supported.])
 
 dnl **** Check for SANE ****
 if test "x$with_sane" != "xno"
diff -durN -x '*~' -x '*.orig' wine-1.7.18.orig/dlls/kernel32/term.c wine-1.7.18/dlls/kernel32/term.c
--- wine-1.7.18.orig/dlls/kernel32/term.c	2014-05-02 20:15:48.000000000 +0200
+++ wine-1.7.18/dlls/kernel32/term.c	2014-05-03 13:16:18.597752974 +0200
@@ -27,6 +27,8 @@
 # include <ncurses.h>
 #elif defined(HAVE_CURSES_H)
 # include <curses.h>
+#elif defined(HAVE_NCURSES_NCURSES_H)
+# include <ncurses/ncurses.h>
 #endif
 /* avoid redefinition warnings */
 #undef KEY_EVENT
@@ -34,6 +36,8 @@
 
 #if defined(HAVE_CURSES_H) || defined(HAVE_NCURSES_H)
 #include <term.h>
+#elif defined(HAVE_NCURSES_NCURSES_H)
+#include <ncurses/term.h>
 #endif
 
 #include <windef.h>
@@ -148,7 +152,7 @@
 
 #if defined(SONAME_LIBCURSES) || defined(SONAME_LIBNCURSES)
 
-#ifdef HAVE_NCURSES_H
+#ifdef HAVE_NCURSES_H || defined(HAVE_NCURSES_NCURSES_H)
 # define CURSES_NAME "ncurses"
 #else
 # define CURSES_NAME "curses"
diff -durN -x '*~' -x '*.orig' wine-1.7.18.orig/programs/wineconsole/curses.c wine-1.7.18/programs/wineconsole/curses.c
--- wine-1.7.18.orig/programs/wineconsole/curses.c	2014-05-02 20:15:48.000000000 +0200
+++ wine-1.7.18/programs/wineconsole/curses.c	2014-05-03 13:16:18.597752974 +0200
@@ -40,6 +40,8 @@
 # include <ncurses.h>
 #elif defined(HAVE_CURSES_H)
 # include <curses.h>
+#elif defined(HAVE_NCURSES_NCURSES_H)
+# include <ncurses/ncurses.h>
 #endif
 /* avoid redefinition warnings */
 #undef KEY_EVENT
@@ -64,7 +66,7 @@
 
 #if defined(SONAME_LIBCURSES) || defined(SONAME_LIBNCURSES)
 
-#ifdef HAVE_NCURSES_H
+#if defined(HAVE_NCURSES_H) || defined(HAVE_NCURSES_NCURSES_H)
 # define CURSES_NAME "ncurses"
 #else
 # define CURSES_NAME "curses"
