diff -dur wine-1.8.orig/dlls/crypt32/rootstore.c wine-1.8/dlls/crypt32/rootstore.c
--- wine-1.8.orig/dlls/crypt32/rootstore.c	2015-12-19 16:57:44.000000000 +0100
+++ wine-1.8/dlls/crypt32/rootstore.c	2016-01-09 18:19:17.673921302 +0100
@@ -490,6 +490,7 @@
  "/usr/local/share/certs/",
  "/etc/sfw/openssl/certs",
  "/etc/security/cacerts",  /* Android */
+ "/etc/certs/ca-certificates.crt",
 };
 
 static const BYTE authenticode[] = {
