--- wine-20040309/documentation/configuring.sgml.orig	2004-03-09 20:52:07.000000000 +0100
+++ wine-20040309/documentation/configuring.sgml	2004-03-15 19:20:18.092682208 +0100
@@ -1701,6 +1701,7 @@
       	The CD-ROM label can be read only if the data track of
       	the disk resides in the first track and the cdrom is
       	iso9660.
+           </para> </listitem>
           <listitem> <para>
       	Support for labels/serial nums WRITING.
             </para> </listitem>
