--- wine-20050524/dlls/winealsa.drv/audio.c.orig	2005-04-27 11:39:56.000000000 +0200
+++ wine-20050524/dlls/winealsa.drv/audio.c	2005-06-06 07:39:42.068122696 +0200
@@ -45,6 +45,9 @@
 # include <sys/mman.h>
 #endif
 #include "windef.h"
+#define ALSA_PCM_NEW_HW_PARAMS_API
+#define ALSA_PCM_NEW_SW_PARAMS_API
+#include "alsa.h"
 #include "winbase.h"
 #include "wingdi.h"
 #include "winerror.h"
@@ -58,9 +61,6 @@
 #include "ks.h"
 #include "ksguid.h"
 #include "ksmedia.h"
-#define ALSA_PCM_NEW_HW_PARAMS_API
-#define ALSA_PCM_NEW_SW_PARAMS_API
-#include "alsa.h"
 #include "wine/library.h"
 #include "wine/unicode.h"
 #include "wine/debug.h"
