diff -dur wine-1.7.18.orig/loader/wine.de.UTF-8.man.in wine-1.7.18/loader/wine.de.UTF-8.man.in
--- wine-1.7.18.orig/loader/wine.de.UTF-8.man.in	2014-05-02 20:15:48.000000000 +0200
+++ wine-1.7.18/loader/wine.de.UTF-8.man.in	2014-05-03 13:26:11.771072964 +0200
@@ -1,14 +1,14 @@
 .\" -*- nroff -*-
-.TH WINE 1 "November 2007" "@PACKAGE_STRING@" "Windows On Unix"
+.TH WINE64 1 "November 2007" "@PACKAGE_STRING@" "Windows On Unix"
 .SH NAME
 wine \- Windows-Programme auf Unix-Systemen ausführen
 .SH ÜBERSICHT
-.BI "wine " Programm
+.BI "wine64 " Programm
 [Argumente ... ]
 .br
-.B wine --help
+.B wine64 --help
 .br
-.B wine --version
+.B wine64 --version
 .PP
 Für das Übergeben von Kommandos an Windows-Programme siehe den
 Abschnitt
diff -dur wine-1.7.18.orig/loader/wine.fr.UTF-8.man.in wine-1.7.18/loader/wine.fr.UTF-8.man.in
--- wine-1.7.18.orig/loader/wine.fr.UTF-8.man.in	2014-05-02 20:15:48.000000000 +0200
+++ wine-1.7.18/loader/wine.fr.UTF-8.man.in	2014-05-03 13:26:36.374405744 +0200
@@ -1,13 +1,13 @@
-.TH WINE 1 "juillet 2013" "@PACKAGE_STRING@" "Windows sur Unix"
+.TH WINE64 1 "juillet 2013" "@PACKAGE_STRING@" "Windows sur Unix"
 .SH NOM
 wine \- exécuter des programmes Windows sur Unix
 .SH SYNOPSIS
-.B wine
+.B wine64
 .IR "programme " [ arguments ]
 .br
-.B wine --help
+.B wine64 --help
 .br
-.B wine --version
+.B wine64 --version
 .PP
 Pour des instructions sur le passage d'arguments aux programmes Windows, veuillez lire la section
 .B
diff -dur wine-1.7.18.orig/loader/wine.man.in wine-1.7.18/loader/wine.man.in
--- wine-1.7.18.orig/loader/wine.man.in	2014-05-02 20:15:48.000000000 +0200
+++ wine-1.7.18/loader/wine.man.in	2014-05-03 13:26:59.134405233 +0200
@@ -1,13 +1,13 @@
-.TH WINE 1 "July 2013" "@PACKAGE_STRING@" "Windows On Unix"
+.TH WINE64 1 "July 2013" "@PACKAGE_STRING@" "Windows On Unix"
 .SH NAME
 wine \- run Windows programs on Unix
 .SH SYNOPSIS
-.B wine
+.B wine64
 .IR "program " [ arguments ]
 .br
-.B wine --help
+.B wine64 --help
 .br
-.B wine --version
+.B wine64 --version
 .PP
 For instructions on passing arguments to Windows programs, please see the
 .B
diff -dur wine-1.7.18.orig/loader/wine.pl.UTF-8.man.in wine-1.7.18/loader/wine.pl.UTF-8.man.in
--- wine-1.7.18.orig/loader/wine.pl.UTF-8.man.in	2014-05-02 20:15:48.000000000 +0200
+++ wine-1.7.18/loader/wine.pl.UTF-8.man.in	2014-05-03 13:27:13.701071570 +0200
@@ -1,14 +1,14 @@
 .\" -*- nroff -*-
-.TH WINE 1 "October 2005" "@PACKAGE_STRING@" "Windows On Unix"
+.TH WINE64 1 "October 2005" "@PACKAGE_STRING@" "Windows On Unix"
 .SH NAZWA
 wine \- uruchamiaj programy Windowsowe na Uniksie
 .SH SKŁADNIA
-.BI "wine " program
+.BI "wine64 " program
 [opcje ... ]
 .br
-.B wine --help
+.B wine64 --help
 .br
-.B wine --version
+.B wine64 --version
 .PP
 Informacji na temat przekazywania opcji programom Windowsowym szukaj w rozdziale
 .B
