--- wine-1.1.36/tools/winegcc/winegcc.c.orig	2010-01-08 20:20:11.000000000 +0100
+++ wine-1.1.36/tools/winegcc/winegcc.c	2010-01-19 00:19:19.000000000 +0100
@@ -289,8 +289,7 @@
         break;
     case proc_cc:
     case proc_as:
-        if (opts->target) str = strmake( "%s-gcc", opts->target );
-        else str = CC;
+        str = CC;
         break;
     case proc_cxx:
         if (opts->target) str = strmake( "%s-g++", opts->target );
