diff -Nur wine-20011108.orig/graphics/x11drv/xfont.c wine-20011108/graphics/x11drv/xfont.c
--- wine-20011108.orig/dlls/winex11.drv/xfont.c	Tue Nov  6 01:46:36 2001
+++ wine-20011108/dlls/winex11.drv/xfont.c	Fri Nov 16 11:29:37 2001
@@ -2191,11 +2191,13 @@
 			       goto fail;
 			   }
 
-			   if( pfi->df.dfPixHeight == 0 )
-			   {
-			       TRACE("error: dfPixHeight==0\n");
-			       goto fail;
-			   }
+// To strict font checking - commented
+// FIXME - adaptarized to 20011108 by pascalek@pld.org.pl
+//			   if( pfi->df.dfPixHeight == 0 )
+//			   {
+//			       TRACE("error: dfPixHeight==0\n");
+//			       goto fail;
+//			   }
 
 			   pfi->df.dfFace = pfr->lfFaceName;
 			   if( pfi->fi_flags & FI_SCALABLE )
