--- wine-1.4.1/configure.ac~	2013-06-13 12:53:42.143966172 +0200
+++ wine-1.4.1/configure.ac	2013-06-13 12:57:28.566381773 +0200
@@ -1267,7 +1267,8 @@
         CPPFLAGS="$CPPFLAGS $ac_gnutls_cflags"
     fi
     AC_CHECK_HEADER(gnutls/gnutls.h,
-        [AC_COMPILE_IFELSE([AC_LANG_PROGRAM([[#include <gnutls/gnutls.h>]], [[static typeof(gnutls_mac_get_key_size) *func; if (func) return 0;]])],
+        [AC_COMPILE_IFELSE([AC_LANG_PROGRAM([[#include <gnutls/gnutls.h>
+#include <gnutls/crypto.h>]], [[static typeof(gnutls_mac_get_key_size) *func; if (func) return 0;]])],
             [WINE_CHECK_SONAME(gnutls,gnutls_global_init,
                 [AC_SUBST(GNUTLSINCL,"$ac_gnutls_cflags")],,[$ac_gnutls_libs])])])
     CPPFLAGS="$ac_save_CPPFLAGS"
--- wine-1.4.1/dlls/secur32/schannel_gnutls.c~	2012-06-15 21:17:00.000000000 +0200
+++ wine-1.4.1/dlls/secur32/schannel_gnutls.c	2013-06-13 13:11:32.414575193 +0200
@@ -25,6 +25,7 @@
 #include <stdarg.h>
 #ifdef SONAME_LIBGNUTLS
 #include <gnutls/gnutls.h>
+#include <gnutls/crypto.h>
 #endif
 
 #include "windef.h"
