--- wine-20030115/configure.ac.orig	Wed Feb 19 18:12:30 2003
+++ wine-20030115/configure.ac	Wed Feb 19 18:37:12 2003
@@ -758,7 +758,7 @@
 dnl **** Check for .string in assembler ****
 
 AC_CACHE_CHECK([whether assembler accepts .string], ac_cv_c_asm_string,
-    WINE_TRY_ASM_LINK([".data\n\t.string \"test\"\n\t.text"],,,
+    WINE_TRY_ASM_LINK([".data\n\t.string \\"test\\"\n\t.text"],,,
                       ac_cv_c_asm_string="yes",ac_cv_c_asm_string="no"))
 if test "$ac_cv_c_asm_string" = "yes"
 then
