--- wine-1.1.28/loader/Makefile.in.orig	2009-08-22 23:32:18.000000000 +0200
+++ wine-1.1.28/loader/Makefile.in	2009-08-22 23:34:32.000000000 +0200
@@ -23,8 +23,8 @@
 INSTALLDIRS = \
 	$(DESTDIR)$(bindir) \
 	$(DESTDIR)$(mandir)/man$(prog_manext) \
-	$(DESTDIR)$(mandir)/de.UTF-8/man$(prog_manext) \
-	$(DESTDIR)$(mandir)/fr.UTF-8/man$(prog_manext)
+	$(DESTDIR)$(mandir)/de/man$(prog_manext) \
+	$(DESTDIR)$(mandir)/fr/man$(prog_manext)
 
 all: wine wine-installed $(EXTRA_BINARIES) $(MANPAGES)
 
@@ -54,13 +54,13 @@
 
 install-man-pages: $(MANPAGES) $(INSTALLDIRS) dummy
 	$(INSTALL_DATA) wine.man $(DESTDIR)$(mandir)/man$(prog_manext)/wine.$(prog_manext)
-	$(INSTALL_DATA) wine.de.man $(DESTDIR)$(mandir)/de.UTF-8/man$(prog_manext)/wine.$(prog_manext)
-	$(INSTALL_DATA) wine.fr.man $(DESTDIR)$(mandir)/fr.UTF-8/man$(prog_manext)/wine.$(prog_manext)
+	$(INSTALL_DATA) wine.de.man $(DESTDIR)$(mandir)/de/man$(prog_manext)/wine.$(prog_manext)
+	$(INSTALL_DATA) wine.fr.man $(DESTDIR)$(mandir)/fr/man$(prog_manext)/wine.$(prog_manext)
 
 uninstall::
 	-cd $(DESTDIR)$(bindir) && $(RM) $(MAIN_BINARY) $(EXTRA_BINARIES)
 	$(RM) $(DESTDIR)$(mandir)/man$(prog_manext)/wine.$(prog_manext)
-	$(RM) $(DESTDIR)$(mandir)/de.UTF-8/man$(prog_manext)/wine.$(prog_manext)
-	$(RM) $(DESTDIR)$(mandir)/fr.UTF-8/man$(prog_manext)/wine.$(prog_manext)
+	$(RM) $(DESTDIR)$(mandir)/de/man$(prog_manext)/wine.$(prog_manext)
+	$(RM) $(DESTDIR)$(mandir)/fr/man$(prog_manext)/wine.$(prog_manext)
 
 @DEPENDENCIES@  # everything below this line is overwritten by make depend
--- wine-1.1.16/server/Makefile.in~	2009-02-27 23:21:46.000000000 +0100
+++ wine-1.1.16/server/Makefile.in	2009-02-28 14:50:16.838539394 +0100
@@ -62,7 +62,7 @@
 INSTALLDIRS = \
 	$(DESTDIR)$(bindir) \
 	$(DESTDIR)$(mandir)/man$(prog_manext) \
-	$(DESTDIR)$(mandir)/fr.UTF-8/man$(prog_manext)
+	$(DESTDIR)$(mandir)/fr/man$(prog_manext)
 
 all: $(PROGRAMS) $(MANPAGES)
 
@@ -77,10 +77,10 @@
 install install-lib:: wineserver-installed $(MANPAGES) $(INSTALLDIRS)
 	$(INSTALL_PROGRAM) wineserver-installed $(DESTDIR)$(bindir)/wineserver
 	$(INSTALL_DATA) wineserver.man $(DESTDIR)$(mandir)/man$(prog_manext)/wineserver.$(prog_manext)
-	$(INSTALL_DATA) wineserver.fr.man $(DESTDIR)$(mandir)/fr.UTF-8/man$(prog_manext)/wineserver.$(prog_manext)
+	$(INSTALL_DATA) wineserver.fr.man $(DESTDIR)$(mandir)/fr/man$(prog_manext)/wineserver.$(prog_manext)
 
 uninstall::
 	$(RM) $(DESTDIR)$(bindir)/wineserver $(DESTDIR)$(mandir)/man$(prog_manext)/wineserver.$(prog_manext)
-	$(RM) $(DESTDIR)$(mandir)/fr.UTF-8/man$(prog_manext)/wineserver.$(prog_manext)
+	$(RM) $(DESTDIR)$(mandir)/fr/man$(prog_manext)/wineserver.$(prog_manext)
 
 @DEPENDENCIES@  # everything below this line is overwritten by make depend
