diff -ur wine-20050830/configure.ac wine-20050830.new/configure.ac
--- wine-20050830/configure.ac	2005-08-25 12:14:13.000000000 +0000
+++ wine-20050830.new/configure.ac	2005-08-30 21:06:27.136642632 +0000
@@ -467,12 +467,12 @@
     then
         AC_CHECK_LIB(ncurses,waddch,
             [AC_DEFINE(HAVE_LIBNCURSES, 1, [Define if you have the ncurses library (-lncurses)])
-             CURSESLIBS="-lncurses"])
+             CURSESLIBS="-lncurses -ltinfo"],
     elif test "$ac_cv_header_curses_h" = "yes"
     then
         AC_CHECK_LIB(curses,waddch,
             [AC_DEFINE(HAVE_LIBCURSES, 1, [Define if you have the curses library (-lcurses)])
-             CURSESLIBS="-lcurses"])
+             CURSESLIBS="-lcurses -ltinfo"])])])])
     fi
 fi
 AC_SUBST(CURSESLIBS)
